README.md

```diff
@@ -1,22 +1,24 @@
+[![Travis](https://img.shields.io/travis/numpy/numpy.svg)](https://travis-ci.org/numpy/numpy) 
+
```

Comment:
You can see the rendering here: https://github.com/techtonik/numpy/blob/patch-2/README.md